VERSION

```diff
@@ -1 +1 @@
-3.2.0-dev
+3.2.0.1
```

Comment:
thanks!

Reply:
Welcome. It's aligned with NCP release version.